build.sh

```diff
@@ -1,12 +1,28 @@
+#!/bin/bash -xv
+
+NODE_CWD=./node_modules/.bin
+
+JSPM_CMD=${NODE_CWD}/jspm
+
+JSPM_VERSION=beta
+REGENERATOR_VERSION=$(npm v regenerator@^0.8.42 version | tail -n 1 | awk '{print $2}' | tr -d "'")
+
+#
+
 cd build-babel
-jspm install --quick --log err
-jspm build systemjs-babel.js ../systemjs-babel-browser.js --format amd --skip-source-maps --skip-rollup --minify --log ok
-jspm build systemjs-babel.js ../systemjs-babel-node.js --format amd --node --skip-source-maps --skip-rollup --log ok
-jspm run systemjs-build-babel-helpers.js > ../babel-helpers.js
-jspm run systemjs-build-modular-babel-helpers.js
+rm -rf jspm_packages node_modules
+
+npm install jspm@${JSPM_VERSION} --cache-min=Infinity
+${JSPM_CMD} install --quick --log err
+cp -r jspm_packages_override jspm_packages
+
+${JSPM_CMD} build systemjs-babel.js ../systemjs-babel-browser.js --format amd --skip-source-maps --skip-rollup --minify --log ok
+${JSPM_CMD} build systemjs-babel.js ../systemjs-babel-node.js --format amd --node --skip-source-maps --skip-rollup --log ok
+${JSPM_CMD} run systemjs-build-babel-helpers.js > ../babel-helpers.js
+${JSPM_CMD} run systemjs-build-modular-babel-helpers.js
 (
   echo "export default (function(module) {" ;
-  cat jspm_packages/npm/regenerator-runtime@0.9.5/runtime.js
+  cat jspm_packages/github/facebook/regenerator@${REGENERATOR_VERSION}/runtime.js
```

Comment:
You're probably right actually
On Thu, 26 May 2016 at 12:27, Douglas Duteil <notifications@github.com>
wrote:

> In build.sh
> <https://github.com/systemjs/plugin-babel/pull/38#discussion_r64723647>:
>
> >  (
> >    echo "export default (function(module) {" ;
> > -  cat jspm_packages/npm/regenerator-runtime@0.9.5/runtime.js
> > +  cat jspm_packages/github/facebook/regenerator@${REGENERATOR_VERSION}/runtime.js
>
> Sorry isn't better to use the NPM version of regenerator ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/systemjs/plugin-babel/pull/38/files/956cd69a4506d50b97dc0b33800393791ba46a22#r64723647>
>